### PR TITLE
Bug 1553734 - Quickly copy full commit hash

### PR DESCRIPTION
**Pull request by ionutgoldan**

### CODE REVIEW

File: ui/job-view/pushes/Revision.jsx [hunk 1]
```diff
@@ -59,6 +60,12 @@ export class Revision extends React.PureComponent {
       <li className="clearfix">
         <span className="revision" data-tags={this.tags}>
           <span className="revision-holder">
+            <CopyToClipboard text={commitRevision}>
+              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+              <a title="Copy full hash">
```
Comment (sarah-clements) [inline ui/job-view/pushes/Revision.jsx]: I'd get rid of the anchor tag and give the `FontAwesomeIcon` or a span a className='pointer' (that class is in the treeherder-global stylesheet). Also, giving the FontAwesomeIcon a title attribute will provide an `aria-labelledby` tag: https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility#svg-semantic

Comment (ionutgoldan) [reply ui/job-view/pushes/Revision.jsx]: Thanks a lot for your suggestions!

File: ui/job-view/pushes/Revision.jsx [hunk 2]
```diff
@@ -59,6 +60,12 @@ export class Revision extends React.PureComponent {
       <li className="clearfix">
         <span className="revision" data-tags={this.tags}>
           <span className="revision-holder">
+            <CopyToClipboard text={commitRevision}>
+              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```
Comment (sarah-clements) [inline ui/job-view/pushes/Revision.jsx]: I think you can remove this disabled linting rule now.

**Comment by ionutgoldan** [comment]
> 
> 
> Looks good, thanks. You'll need to include the `yarn.lock` file changes in this pr to get the tests to pass (this is automatically generated when you use `yarn add <package to install>`).
> 
> Edit: On second thought, we don't need the `react-copy-to-clipboard` package. There's a native [clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard) that I used for the alerts view: https://github.com/mozilla/treeherder/blob/master/ui/perfherder/alerts/StatusDropdown.jsx#L101
> 
> It'd be preferable to use this instead since we have a large bundle size already.

👍

**Comment by armenzg** [comment]
In production.

**Pull request merged**